includes/admin/class.llms.admin.notices.php

```diff
@@ -272,7 +272,10 @@ protected static function load_notices() {
 			$option = array( $option );
 		}
 
-		return array_filter( array_filter( $option, 'is_string' ) );
+		// Remove empty and non-string values.
+		return array_filter( function ( $option ) {
+			return ( ! empty( $option ) && is_string( $option ) );
+		} );
```

Comment:
```suggestion
		return array_filter( $option, function ( $opt ) {
			return ( ! empty( $opt ) && is_string( $opt ) );
		} );
```
(the inner var change is for readability)